src/bin/winkernel/msquicpriv.kernel.vcxproj

```diff
@@ -116,12 +116,12 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
     <ClCompile>
-      <PreprocessorDefinitions>QUIC_EVENTS_MANIFEST_ETW;QUIC_LOGS_MANIFEST_ETW;SECURITY_KERNEL;SECURITY_WIN32;_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Do we need to define this in all the other kernel projects too? Otherwise the base static libraries won't have this defined, which is where we want that.

Reply:
I was thinking about it, but I think this is good for now. This is at least 98% of what we want.

Reply:
Updated core now.